### ISSUE TITLE
byobu: open terminal failed: not a terminal

### ISSUE THREAD

**Issue by mvysny** [termux/termux-packages#2979]
On a fresh installation of termux (no custom `.bashrc`, nothing, completely fresh installation of termux), byobu fails to start with the following message:
```
$ byobu
ifconfig : error fetching interface information: Device not found
open terminal failed: not a terminal
```
However, `screen` starts properly. This is not over a ssh - I just type `byobu` straight to the phone virtual keyboard.

This happens on Android One 8.1.0 running on Mi A2 Lite. Thank you :)

**Comment by mvysny** [comment]
![screenshot_20181023-084310](https://user-images.githubusercontent.com/254091/47338506-d5825500-d6a0-11e8-9eb3-262ee2e3fad8.png)

**Comment by its-pointless** [comment]
This also happens on occation with tmux. But you get the same state everytime witu byobu. It only affects later versions of android.

**Comment by its-pointless** [comment]
Okay i got it working and have pull request #2980.

**Comment by fornwall** [comment]
It turns out that this was actually an issue caused by `gawk` (which byobu uses), which has been fixed in version `4.2.1-1` of that package. Run `pkg up` to update!

**Comment by mvysny** [comment]
Thank you, I confirm that byobu is now working, thank you! :)
